phpunit.xml.dist

```diff
@@ -23,10 +23,10 @@
         <env name="HTTP_USERNAME" value="admin"/>
         <env name="HTTP_PASSWORD" value="changeit"/>
         <!-- for postgres testing -->
-        <env name="PG_HOST" value="127.0.0.1"/>
+        <env name="PG_HOST" value="postgres"/>
```

Comment:
don't edit this here, copy to phpunit.xml instead (and put to .gitignore), change these values only for yourself.

Reply:
I will enable travis with postgres directly, no need for docker on travis